nixos/modules/services/web-servers/traefik.nix

```diff
@@ -24,38 +23,35 @@ in {
       default = null;
       example = /path/to/config.toml;
       type = types.nullOr types.path;
-      description = "Verbatim traefik.toml to use";
+      description = ''
+        Path to verbatim traefik.toml to use.
+        (Using that option has precedence over <literal>configOptions</literal>)
+      '';
     };
+
     configOptions = mkOption {
       description = ''
         Config for Traefik.
       '';
       type = types.attrs;
+      default = {
+        defaultEntryPoints = ["http"];
+        entryPoints.http.address = ":80";
```

Comment:
Also this configuration does nothing it is good to have a nice to have a running example by default (Even if it just for testing new versions of traefik).